tests/test_basics.py

```diff
@@ -800,3 +800,11 @@ def test_last_event_id_transaction(sentry_init):
         pass
 
     assert last_event_id() is None, "Transaction should not set last_event_id"
+
+
+def test_last_event_id_scope(sentry_init):
+    sentry_init(enable_tracing=True)
+
+    # Should not crash
+    with push_scope() as scope:
```

Comment:
```suggestion
    with isolation_scope() as scope:
```

`push_scope` is [deprecated](https://github.com/getsentry/sentry-python/blob/master/CHANGELOG.md#deprecated) since version 2.0.0, so let's use the new `isolation_scope` function here, instead. Using `isolation_scope` also fails on master but passes on this branch.